### PR TITLE
Save the module active scalars by name in the state file

### PR DESCRIPTION
**Pull request by alesgenova**

### CODE REVIEW

**Comment by psavery** [comment]
I haven't tried this out, but I think it looks good to me.

File: tomviz/modules/Module.h [hunk 1]
```diff
@@ -114,7 +114,9 @@ class Module : public QObject
 
   /// Returns the active scalars of the module
   int activeScalars() const { return m_activeScalars; }
-  static const int DEFAULT_SCALARS;
+  QString activeScalarsName() const;
+  static const QString DEFAULT_SCALARS_NAME;
```
Comment (cryos) [inline tomviz/modules/Module.h:118]: Can we not use macro style names, I think using s_defaultScalarsName and s_defaultScalarsIdx would be preferable.

File: tomviz/DataSource.h [hunk 2]
```diff
@@ -234,6 +234,8 @@ class DataSource : public QObject
   int activeScalarsIdx() const;
   /// Get the scalars name for a given index.
   QString scalarsName(int arrayIdx) const;
+  /// Get the scalars idx for a given name.
```
Comment (cryos) [inline tomviz/DataSource.h:237]: Leave a blank line before new function/docs

**Comment by cryos** [comment]
Can you just fix the DCO error, then I think we can merge.

**Pull request merged**